config/initializers/config.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+Config.setup do |config|
+  # Name of the constant exposing loaded settings
+  config.const_name = 'Settings'
+
+  # Ability to remove elements of the array set in earlier loaded settings file. For example value: '--'.
+  #
+  # config.knockout_prefix = nil
+
+  # Overwrite an existing value when merging a `nil` value.
+  # When set to `false`, the existing value is retained after merge.
+  #
+  # config.merge_nil_values = true
+
+  # Overwrite arrays found in previously loaded settings file. When set to `false`, arrays will be merged.
+  #
+  # config.overwrite_arrays = true
+
+  # Defines current environment, affecting which settings file will be loaded.
+  # Default: `Rails.env`
+  #
+  # config.environment = ENV.fetch('ENVIRONMENT', :development)
+
+  # Load environment variables from the `ENV` object and override any settings defined in files.
+  #
+  # config.use_env = false
+
+  # Define ENV variable prefix deciding which variables to load into config.
+  #
+  # Reading variables from ENV is case-sensitive. If you define lowercase value below, ensure your ENV variables are
+  # prefixed in the same way.
+  #
+  # When not set it defaults to `config.const_name`.
+  #
+  config.env_prefix = 'SETTINGS'
```

Comment:
Why are we setting this if we aren't loading from the environment (line 27)?

Reply:
These are the default settings from the gem: https://github.com/rubyconfig/config/blob/master/lib/generators/config/templates/config.rb